src/checkUrl.js

```diff
@@ -0,0 +1,34 @@
+const cases = [
+  'SNAKE',
+  'KEBAB',
+  'CAMEL',
+  'PASCAL',
+  'UPPER',
+];
```

Comment:
Usually, we move such constants to separate files and use uppercased names for them